### PR TITLE
Create community.md

### PR DESCRIPTION
**Pull request by ShanEllis**

### CODE REVIEW

File: _drafts/community.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  - name: Shannon E. Ellis
+    url: https://shanellis.github.io/
+categories:
+  - blog
+topicid: ???
+tags:
+  - unconf
+  - community
+  - meetings
+---
```
Comment (stefaniebutland) [inline _drafts/community.md]: @ShanEllis thanks for the post. @karthik and I will give comments soon and hope to post Fri June 23

Comment (stefaniebutland) [inline _drafts/community.md]: @sckott feel free to comment as well, but no obligation

File: _drafts/community.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+What's that? You've heard of R? You use R? You develop in R? You know someone else who's mentioned R? Oh, you're breathing? Well, in that case, welcome! Come join the R community!
+
+We recently had a group discussion at [rOpenSci](https://ropensci.org/)'s  [#runconf17](https://twitter.com/search?q=%23runconf17&lang=en) in Los Angeles, CA about the R community. I initially opened the [issue on GitHub](https://github.com/ropensci/unconf17/issues/63). After this issue was well-received (check out the emoji-love below!), we realized people were keen to talk about this and decided to have an optional and informal discussion in person.
+
+<center>![](https://ShanEllis.github.io/images/github_feedback.png)</center>
```
Comment (karthik) [inline _drafts/community.md]: Replace with this:

```
<center>
![](https://ShanEllis.github.io/images/github_feedback.png)
</center>
```

File: _drafts/community.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+## Improving inclusivity
+
+No community is perfect, and being willing to consider our shortcomings and think about ways in which we can improve is so important. The group came up with a lot of great suggestions, including many I had not previously thought of personally.
+
+[Alice Daish](https://twitter.com/alice_data) did a great job capturing the conversation and allowing for more discussion online:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Join the lunchtime <a href="https://twitter.com/hashtag/runconf17?src=hash">#runconf17</a> discussion about the <a href="https://twitter.com/hashtag/rstats?src=hash">#rstats</a> communities - what do we need to do to improve? <a href="https://t.co/ztbXxNfqU7">pic.twitter.com/ztbXxNfqU7</a></p>&mdash; Alice Data (@alice_data) <a href="https://twitter.com/alice_data/status/868198529134510080">May 26, 2017</a></blockquote>
+
+To summarize here:
+
```
Comment (stefaniebutland) [inline _drafts/community.md]: @ShanEllis While this didn't come up at the discussion, might be good to add to your bullet list (could just add a text line indicating David Smith suggested it, not embed the tweet)
https://twitter.com/revodavid/status/874687595120406528

This hasn't been adopted (that would be a whole other project), but adding it here might get a couple of eyes on it.

File: _drafts/community.md (hunk 1 above)
Comment (stefaniebutland) [inline _drafts/community.md]: @ShanEllis Karthik says it's a great post and we'd love to make it live this Friday June 23! (might go up Thurs night). His comment below shows how to make the emoji pic show up.

Please tag me and @sckott when you've made that change and consider the one comment below. Scott will merge your pull request and make it live and I'll tweet about it Friday morning!
Thank you so much for taking the initiative on this and "making it real" by doing the extra work a post requires. Now you can get a little twitter recognition for your effort :-)

Comment (ShanEllis) [inline _drafts/community.md]: Thanks @stefaniebutland and @sckott for your help! Comments addressed -- feel free to change wording on the additions (or anything really!)

File: _drafts/community.md (hunk 3 above)
Comment (ShanEllis) [inline _drafts/community.md]: added -- I think the hashtag idea and including it in this post are both great ideas!

**Pull request merged**